data/hai/hai reveal 1 intro.txt

```diff
@@ -524,12 +532,26 @@ mission "Hai Leaks Response 1"
 			`	You haven't really seen very much of the Hai so far, and you can't help but think the size of the detachment is rather militant of them. It's impossible to judge if it's for your safety or whether, given your status as a "war hero", it's for theirs.`
 			`	Word of your status in human space also seems to have spread ahead of you, perhaps because you are a significant human visitor in the current climate. A small crowd of Hai gather to peek at you on the short walk from the spaceport to the government building you are led to.`
 			label resume
+			choice
+				`	(Keep silent and observe.)`
+					goto observe
+				`	(Ask about the large security detachment.)`
```

Comment:
Any choices in parentheses are "action choices," with choices in quotes being "dialog choices." We typically should not have action choices where the action is to say something, such as in this second choice.

Reply:
I reworked that section, so you actually ask about a security detachment. I applied lots of other changes from @bene-dictator to clean up that area too.